src/remote_common.c

```diff
@@ -186,7 +187,11 @@ bool wait_response_message(amqp_connection_state_t *conn,
      * AMQP_FRAME_BODY*/
     for (;;) {
         amqp_maybe_release_buffers(*conn);
-        amqp_simple_wait_frame(*conn, &frame);
+
+        struct timeval t = {10, 0}; /* RPC timeout: 10s*/
```

Comment:
Just curious why 10 seconds is chosen. And maybe these could be macros?

Reply:
It can be changed to other values to make sense and I don't know much about that. 
For consistent, MACRO does not be used.